src/libNode/FinalBlockProcessing.cpp

```diff
@@ -888,7 +888,7 @@ bool Node::ProcessFinalBlock(const vector<unsigned char> & message, unsigned int
     if(m_state == MICROBLOCK_CONSENSUS)
     {
         unsigned int time_pass = 0;
-        while(m_state != PROCESS_FINALBLOCK)
+        while(m_state != WAITING_FINALBLOCK)
```

Comment:
❔ Why did this change? It seems unrelated to the rest of this PR.

Reply:
This was a bug that had to be corrected.